ophyd/epics_motor.py

```diff
@@ -129,6 +146,93 @@ def position(self):
         '''
         return self._position
 
+    @property
+    @raise_if_disconnected
+    def offset(self):
+        '''The current user offset value of the motor in its engineering units
+
+        Returns
+        -------
+        offset : float
+        '''
+        return self.user_offset.get()
+
+    @property
+    @raise_if_disconnected
+    def offset_frozen(self):
+        '''Whether or not the motor offset is frozen
+
+        Returns
+        -------
+        frozen : bool
+        '''
+        return bool(self.motor_offset_freeze_switch.get())
+
+    @property
+    @raise_if_disconnected
+    def at_high_limit_switch(self):
```

Comment:
I am also against using properties where we can use a soft-signal instead for exactly the reason @klauer said.  Properties can not be exposed to `read`.